packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js

```diff
@@ -6,11 +6,15 @@ import map from 'lodash/map';
 import startsWith from 'lodash/startsWith';
 import { LANGUAGES } from '@ovh-ux/manager-config';
 import {
-  PHONE_PREFIX,
-  MODEL_DEBOUNCE_DELAY,
   FIELD_NAME_LIST,
+  MODEL_DEBOUNCE_DELAY,
+  PHONE_PREFIX,
 } from '../new-account-form-component.constants';
 
+const a = 1;
+const b = a + FIELD_NAME_LIST.address;
+const c = `${a + FIELD_NAME_LIST.address}À`;
```

Comment:
**nitpick:** Looks like an unnecessary change

Reply:
ah my bad, not required.
I was made some test and it was committed within.